src/WebJobs.Script.WebHost/Configuration/AzureMonitorLoggingOptionsSetup.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Configuration
+{
+    public class AzureMonitorLoggingOptionsSetup : IConfigureOptions<AzureMonitorLoggingOptions>
+    {
+        private readonly IEnvironment _env;
+
+        public AzureMonitorLoggingOptionsSetup(IEnvironment env)
```

Comment:
Use `IConfiguration` instead, it will include environment variables in it (and is a replacement for `IEnvironment`)